### ISSUE TITLE
Elements shifted off the page with negative locations are considered visible

### ISSUE THREAD

**Issue by twalpole** [teampoltergeist/poltergeist#813]
When using poltergeist (and capybara-webkit) 

    <input id="radio_1" name="radio" value="1" type="radio", style="position:absolute; left: -9999px>

is considered a visible element, although it cannot be seen or interacted with (clicked, etc).  Current selenium-webdriver (2.53.4) reports it as non-visible, although the 3.0.0.beta3.1 version reports it as visible but throws an ElementNotVisible error if you try to click it.    @route - thoughts?

**Comment by route** [comment]
Sorry for a long delay, just found it. @twalpole isn't it a PhantomJS issue?

**Comment by twalpole** [comment]
@route yes it probably is a PhantomJS issue, but it's one we could probably implement a workaround for in Poltergeist.
